.travis.yml

```diff
@@ -6,7 +6,11 @@ install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/ma
 script: bash -ex ./.travis-docker.sh
 env:
   global:
-  - PACKAGE="io-page"
+  - PINS="io-page:. io-page-unix:. io-page-xen:."
   matrix:
-  - DISTRO="alpine-3.5"    OCAML_VERSION=4.02.3 DEPOPTS="mirage-xen-ocaml"
-  - DISTRO="debian-stable" OCAML_VERSION=4.03.0 DEPOPTS="mirage-xen-ocaml"
+  - DISTRO="alpine-3.5"    OCAML_VERSION=4.02.3 PACKAGE="io-page"
```

Comment:
I think bump this test up to 4.04.1 instead of 4.02.3

Also is it worth still supporting 4.02.3 with jbuilder, or bump minimum version to 4.03.0 (which it needs for e.g. ppx_sexp_conv and others anyway).